inst/rmarkdown/templates/nhsr-presentation/skeleton/skeleton.Rmd

```diff
@@ -5,7 +5,22 @@ author: "Author Name"
 institute: "NHS Organisation"
 date: "Presentation Date"
 output:
-  nhsrtheme::nhsr_presentation: default
+  xaringan::moon_reader:
```

Comment:
This isn't going to work outside of the package development, I don't think. The css/ folder won't be visible, as it will be tucked away in the package install location. xaringan::moon_reader function has some tricks to deal with this. I've taken a similar approach in using the nhsrtheme::nhsr_presentation function to find these

Reply:
should note: nhsr_presentation is just a wrapper around xaringan::moon_reader to fix the options e.g. css

Reply:
Ok, thanks for explaining. I'll make the switch!